### PR TITLE
Recycler EMAG Rework

### PR DESCRIPTION
**Pull request by Zap527**
## About the PR
The recycler now deals 30 (60) damage instead of gibbing you when emagged

## Why / Balance
Rules become mechanics.
Emags are incredibly common items which have so many uses it's almost kneecapping yourself NOT to buy one. Since it's one click to make the recycler into a killing machine, it results in many people being gibbed, which is against server rules.
This retains the danger of the recycler and still allows you to make it into a death trap (with more effort), but avoids situations where people would emag the recycler just because (tm) and RR 3 people

## Technical details
Calls TryChangeDamage instead of doing a whole bunch of gibbing stuff. Added a new property to the Recycler component to control the damage output
Logs and comments updated to reflect the change.
Currently the recycler grinds every entities (even items) twice in quick succession, so mobs get dealt the damage twice too. Fixing this is outside of what this PR is trying to accomplish, though.

## Media
https://github.com/user-attachments/assets/7b4cc715-22ed-4ade-bbc1-3d6ef619efe1

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl: Jajsha
- tweak: The emagged recycler now deals massive brute damage instead of instantly gibbing players.

### CODE REVIEW

**Comment by DieselMohawk** [comment]
Looking good

**Comment by DieselMohawk** [comment]
Actually, can you increase the damage to at least crit?

**Comment by DieselMohawk** [comment]
It should be able to instantly crit to prevent meta checks

Example: Security Officer walks into disposals, walks into the grinder, comes out bloody, calls the detective for the "Subverted" recycler.

**Comment by RotEmpress** [comment]
I'm sorta in agreement with Diesel. I genuinely wanna still see the recycler be used for creative/antag use. Completely taking away the damage, sorta just kills the reason to attempt to plan around it.

Jumping into disposals should be a risk, imo!

File: Resources/Prototypes/Entities/Structures/Machines/recycler.yml [hunk 1]
```diff
@@ -84,6 +84,10 @@
       params:
         volume: -3
     cutOffSound: false
+    damage: # Harmony
+      types:
+        Slash: 15
```
Comment (spanky-spanky) [inline Resources/Prototypes/Entities/Structures/Machines/recycler.yml:89]: Increase to 25?

Comment (spanky-spanky) [inline Resources/Prototypes/Entities/Structures/Machines/recycler.yml:89]: Increase to 25?

File: Resources/Prototypes/Entities/Structures/Machines/recycler.yml [hunk 2]
```diff
@@ -84,6 +84,10 @@
...
+        Piercing: 15
```
Comment (spanky-spanky) [inline Resources/Prototypes/Entities/Structures/Machines/recycler.yml:90]: Ditto, 25.

**Review by spanky-spanky — CHANGES_REQUESTED** [review]
Increasing each to 25 with the double pass will do 100 damage, instant crit, preventing meta checks and more lethal but still very revivable.

**Review by FluffMe — DISMISSED** [review]
Code is good. Will merge once you agree on numbers.

**Review by spanky-spanky — APPROVED** [review]

**Pull request merged**